packages/js/src/plugins/candyMachineModule/helpers.ts

```diff
@@ -33,7 +33,7 @@ export function parseCandyMachineItems(rawData: Buffer): CandyMachineItem[] {
 export function getCandyMachineAccountSizeFromData(data: CandyMachineData) {
   if (data.hiddenSettings != null) {
     return CONFIG_ARRAY_START;
-  } else {
+  } 
```

Comment:
I like this change but the next part of the block needs to be indented correctly.

Reply:
not sure about it. Indentation looks ok rn

Reply:
That indentation looks off to me. See screenshot:

![CleanShot 2022-09-13 at 23 28 02@2x](https://user-images.githubusercontent.com/3642397/190020545-bf600fea-1a05-485a-accc-fc5eb2be585a.png)